plane/source/docs/airspeed.rst

```diff
@@ -173,6 +173,9 @@ I2C
 DroneCAN
 --------
 
+- AUAV (static and dynamic pressure)
```

Comment:
does this use a std sensor, like from Amphenol? then a separate caterory naming the sensor should be used...unless AUAV markets a new chip of their own in which case we would need to also link the chip datasheet also

Reply:
I tried to follow the existing structure, for example the next item is DVLR sensors.. So I made a new category with AUAV. Where would you suggest a new header should be?